eng/pipelines/templates/stages/archetype-java-release-pom-only.yml

```diff
@@ -63,6 +63,10 @@ stages:
               runOnce:
                 deploy:
                   steps:
+                    - download: none
```

Comment:
Did you verify if this is required if we explicitly list the download artifacts?

Reply:
I'm checking functionality with -download: none first.   If it's all good, I'll try removing it.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1229360&view=logs&s=69878f40-1e39-53f0-a439-635d139ede3d

Reply:
I verified that adding an explicit download step removes the implicit step.  There's no need to add `download: none` before `download: current`